presto-native-execution/presto_cpp/main/PrestoServer.h

```diff
@@ -89,8 +89,9 @@ class PrestoServer {
 
   virtual std::shared_ptr<velox::exec::ExprSetListener> getExprSetListener();
 
-  /// Returns statistics based http filter.
-  virtual std::unique_ptr<proxygen::RequestHandlerFactory> getHttpStatsFilter();
+  /// Returns any additional http filters.
+  virtual std::vector<std::unique_ptr<proxygen::RequestHandlerFactory>>
+  getAdditionalHttpStatsFilters();
```

Comment:
`s/getAdditionalHttpStatsFilters/getHttpStatsFilters`

By default we are not adding any filter. When the config is given, we get a list of filters and add them. So the name is more suitable with getHttpStatsFilters